applications/serial_lte_modem/src/slm_util.c

```diff
@@ -221,6 +222,49 @@ int util_str_to_int(const char *str_buf, int base, int *output)
 	}
 
 	*output = temp;
+	return 0;
+}
+
+/**
+ * @brief Resolve remote host by hostname or IP address
+ */
+#define SIN6(A) ((struct sockaddr_in6 *)(A))
+#define SIN(A)  ((struct sockaddr_in *)(A))
+
+int util_resolve_host(const char *hostname, uint16_t port, int family, struct sockaddr *sa)
+{
+	int err;
+	struct addrinfo *ai = NULL;
+	struct addrinfo hints = {
+		.ai_family = family
+	};
+
+	if (sa == NULL) {
+		return DNS_EAI_AGAIN;
+	}
+
+	if (family != AF_UNSPEC) {
```

Comment:
Hmm, isn't it just possible to provide hints in any case with family set to `AF_UNSPEC`? It should return either IPv4 or 6 in this case. Or is there some bug in nrf_modem I forgot about?

Reply:
It's not an issue with nrf_modem any more (after fix my JIRA case).
Anyway now `hints` is always required for `ai_flags`